package.json

```diff
@@ -18,7 +18,8 @@
     "test:browser": "run-p test:server test:puppeteer",
     "test:server": "node ./testing/browser/server.js",
     "test:puppeteer": "node ./testing/browser/puppeteer.js",
-    "test:jest": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --verbose=false ./testing/node/"
+    "test:jest": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --verbose=false ./testing/node/",
+    "test:manual": "http-server -o testing/manual/index.html"
```

Comment:
This file appears to be missing, or is that where you're supposed to add your manual tests yourself? If so, maybe a placeholder would be nice?